momento-sdk/src/intTest/java/momento/sdk/TopicClientTest.java

```diff
@@ -61,6 +62,16 @@ public void onCompleted() {
       public void onError(Throwable t) {
         logger.info("onError Invoked");
       }
+
+      @Override
+      public void onDiscontinuity(TopicDiscontinuity discontinuity) {
+        logger.info("onDiscontinuity Invoked");
+      }
+
+      @Override
+      public void onHeartbeat() {
+        logger.info("onHeartbeat Invoked");
+      }
```

Comment:
This could make the test logs pretty chatty. I'd either comment these out or make them debug/trace level.